### PR TITLE
[ie/pornhub] remove phantomjs, update tests

### PR DESCRIPTION
**Pull request by feederbox826**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

Changes:
- remove PhantomJS and watch it be perfectly fine

Tests Changes:
- `jenny-blighe` was removed from the platform, replaced with popular, active pornstar
- videos were hidden from `playlist/44121572`, hence the decrease in count
- md5 would not match before any changes implemented, so changed to match

Tests:
- Private playlist test was successful
- NOT TESTED AGIANST PREMIUM ACCOUNT

Fixes # N/A

Related: https://github.com/yt-dlp/yt-dlp/issues/1354

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

File: yt_dlp/extractor/pornhub.py [hunk 1]
```diff
@@ -464,7 +450,7 @@
         model_profile = self._search_json(
             r'var\s+MODEL_PROFILE\s*=', webpage, 'model profile', video_id, fatal=False)
         video_uploader = self._html_search_regex(
-            r'(?s)From:&nbsp;.+?<(?:a\b[^>]+\bhref=["\']/(?:(?:user|channel)s|model|pornstar)/|span\b[^>]+\bclass=["\']username)[^>]+>(.+?)<',
+            r'<script type="application\/ld\+json">(?:.|\n)+"author" : "(.+)",',
```
[the remark this answers is not in the document]
Comment (feederbox826) [reply yt_dlp/extractor/pornhub.py]: Exclude video_uploader lookup entirely and rely on preexisting ldjson merge

**Comment by bashonly** [comment]
Aside from updating the tests, I don't understand why the changes are being made. Can you please give your rationale behind removing the PhantomJS code and the `uploader` field?

**Comment by feederbox826** [comment]
The `uploader` field is not removed, my understanding was that it would be pulled from the ld+json in `merge_dicts`.

My understanding of PhantomJS and the issues around it is that it should be avoided or removed if possible and my tests on different IPs don't yield the browser verification

**Comment by bashonly** [comment]
PhantomJS can be avoided on a per-user basis by the user simply not installing it. AFAIK the verification is still required, particularly during playlist downloads when some sort of rate-limit has been exceeded

What is actually served in the JSON-LD can vary by region and we should not rely solely on it if we have an alternate source for a metadata value

**Comment by feederbox826** [comment]
will pr test changes then

**Pull request closed without merging**